### PR TITLE
Update App.css

### PR DESCRIPTION
**Pull request by roundband5**
new awesome color 💯

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -14,14 +14,14 @@
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: #cc338b;
```
Comment (lchua2314) [inline src/App.css:17]: Why?

File: src/App.css [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
   min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  color: pink;
```
Comment (lchua2314) [inline src/App.css:24]: Not our theme. Please change.

**Review by leonzh2k — APPROVED** [review]
Awesome!

**Review by lchua2314 — CHANGES_REQUESTED** [review]

File: src/App.css [hunk 3]
```diff
@@ -14,14 +14,14 @@
...
-  background-color: #282c34;
+  background-color: #cc338b;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  color: black;
```
Comment (lchua2314) [inline src/App.css:24]: Why? Approved.

**Review by lchua2314 — APPROVED** [review]

**Comment by lchua2314** [comment]
Looks good, now...

**Pull request merged**